### PR TITLE
Create UiRenderer class

### PR DESCRIPTION
**Pull request by NicholasTanYY**
Handles the ASCII art display of week days and header.

### CODE REVIEW

File: src/main/java/ui/UiRenderer.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+// uncomment below line when merging with Taskmanager
+// import main.TaskManager;
+
+public class UiRenderer {
+    private static final String[] WEEK_DAYS = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
+
+    public static void printWeekHeader() {
+        for (String day : WEEK_DAYS) {
+            System.out.print("+------------");
```
Comment (clarencepohh) [inline src/main/java/ui/UiRenderer.java:15]: Borders for the calendar could be replaced with String literals for better code clarity! LGTM

**Review by clarencepohh — COMMENTED** [review]
Overall looks good! Let's settle the dependency matters once all the required classes are committed 👍

File: src/main/java/ui/UiRenderer.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+    public static void printWeekDays(LocalDate startOfWeek, DateTimeFormatter dateFormatter, TaskManager taskManager) {
+        LocalDate date = startOfWeek;
+        for (int i = 0; i < 7; i++) {
+            System.out.printf("| %-10s ", dateFormatter.format(date));
+            date = date.plusDays(1);
+        }
+        System.out.println("|");
+
+        for (String day : WEEK_DAYS) {
```
Comment (Mahesh1772) [inline src/main/java/ui/UiRenderer.java:38]: Maybe define this as a constant for future use cases!

File: src/main/java/ui/UiRenderer.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+
+// uncomment below line when merging with Taskmanager
+// import main.TaskManager;
+
+public class UiRenderer {
+    private static final String[] WEEK_DAYS = {"Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"};
```
Comment (Mahesh1772) [inline src/main/java/ui/UiRenderer.java:11]: We could perhaps use the built in capabilities of LocalDate package for the weekdays?

File: src/main/java/ui/UiRenderer.java (hunk 1 above)
Comment (NicholasTanYY) [reply src/main/java/ui/UiRenderer.java:15]: Well said, I will take note to include a macro in the next code iteration!

File: src/main/java/ui/UiRenderer.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
+package ui;
```
Comment (Mahesh1772) [inline src/main/java/ui/UiRenderer.java:1]: Overall nice abstraction and neat code!

**Review by Mahesh1772 — COMMENTED** [review]
Good quality code which could use some help.

File: src/main/java/ui/UiRenderer.java (hunk 3 above)
Comment (kyhjonathan) [inline src/main/java/ui/UiRenderer.java:11]: Good usage of final to prevent any accidental changes to WEEK_DAYS!

**Review by kyhjonathan — COMMENTED** [review]
LGTM! Overall neat and understandable code. Shows SLAP code!

**Pull request merged**